src/main.zig

```diff
@@ -609,10 +669,48 @@ const ServerCertificate = struct {
     bytes: []const u8,
     dn: []const u8,
     common_name: []const u8,
+    raw_subject_alternative_name: []const u8,
```

Comment:
I wanted to pre-parse this, but unfortunately that requires adding a new allocation (or a large enough buffer) so this adopts an iterator approach instead.

Reply:
Also this name sucks